assets/js/components/OptIn.js

```diff
@@ -72,17 +72,21 @@ export default function OptIn( { id, name, className, optinAction } ) {
 				disabled={ saving }
 				onChange={ handleOptIn }
 			>
-				{ createInterpolateElement(
-					__( 'Help us improve the Site Kit plugin by allowing tracking of anonymous usage stats. All data are treated in accordance with <a>Google Privacy Policy</a>', 'google-site-kit' ),
-					{
-						a: <Link
-							key="link"
-							href={ 'https://policies.google.com/privacy' }
-							external
-							inherit
-						/>,
+				<span>{ __( 'Help us improve Site Kit by sharing anonymous usage data.', 'google-site-kit' ) }&nbsp;</span>
```

Comment:
We use `{ ' ' }` for a single space instead of `&nbsp;`:
```suggestion
				<span>{ __( 'Help us improve Site Kit by sharing anonymous usage data.', 'google-site-kit' ) }{ ' ' }</span>
```